UnitySDK/Assets/ML-Agents/Examples/Soccer/Scripts/AgentSoccer.cs

```diff
@@ -3,6 +3,13 @@
 
 public class AgentSoccer : Agent
 {
+    // Note that that the detectable tags are different for the blue and purple teams. The order is
```

Comment:
Not sure where else to document this. It's a bit non-intuitive if you just look at the tags in the sensor.

Reply:
seems pretty self documenting in the editor.  Having it here only makes it more clear.